eland/common.py

```diff
@@ -84,6 +86,24 @@ def from_string(order: str) -> "SortOrder":
         return SortOrder.DESC
 
 
+def datetime_to_elasticsearch_date(value: datetime) -> str:
```

Comment:
I don't think this is needed. See below comment

Reply:
How do you want this to be implemented? Calling `strftime` in every comparison would be kinda silly. This is why I created a  common method for the string conversion.

Reply:
It's only called once per API request when the body of `search()` is built. I actually like keeping the `datetime` object as itself here so it can't be confused with a string being passed to the `Filter`. Thoughts on this?

Reply:
I see. It was suitable to do the conversion in the comparison methods, because they handle data types using `isinstance`, but your suggestion seems better. It should also make the comparison methods less bulky.

Can you give me some information where the conversion to string/date formatting should be performed?

Reply:
The conversion happens in the Elasticsearch client :) So you don't have to worry about handling the serialization. Give it a try!

Reply:
Done :)